test/functional/wallet_listtransactions.py

```
@@ -97,5 +97,17 @@ def run_test(self):
                                 {"category": "receive", "amount": Decimal("0.1")},
                                 {"txid": txid, "label": "watchonly"})
 
+        self.test_op_return()
+
+    def test_op_return(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_op_return`

Add return type annotation: `None`

(ANN201)